lib/foreman_tasks/tasks/dynflow.rake

```diff
@@ -1,5 +1,5 @@
 namespace :dynflow do
-  task :client do
+  task :client => :environment do
```

Comment:
@adamruzicka any thoughts about this?

Reply:
Droping it and regenerating `.rubocop_todo.yml`.
It doesn't belong to this PR. :)